src/components/Homepage/Article.js

```diff
@@ -13,7 +27,7 @@ function Article({username,image,timeStamp,likes,title,description,tags}) {
             <p className="text-grey-link text-xs">{new Date(timeStamp).toDateString()}</p>
           </div>
         </div>
-        <button className="flex border justify-center items-center min-w-12 min-h-8 w-12 h-8 border-main-green text-main-green hover:bg-main-green hover:text-red-500">
+        <button onClick={handleLike} className={isLiked ? likedButtonStyle : unlikedButtonStyle}>
```

Comment:
Since the hearting has mostly isolated logic, lets move it to its own component. Lets also make it generic so it can work for anything not just articles. (pass in props to make it work in the article component for example.

Reply:
I created a component named LikeButton: `<LikeButton handleLike={} likes={} isLiked={} />` that accepts the props: `handleLike: () => void`, `likes: number`, `isLiked: boolean`.